frontend/src/config/apis/endpoint.ts

```diff
@@ -0,0 +1,39 @@
+import { HEADERS } from "./header";
+
+export const API_URLS = {
+  ROBOT: {
+    sayHello: () => ({
+      endPoint: "/api/v1/robot/hello",
+      method: "GET",
+      header: HEADERS.header(),
+    }),
+
+    registerRobot: () => ({
+      endPoint: "/api/v1/ui/register",
+      method: "GET",
+      header: HEADERS.header(),
+    }),
+
+    startPatrol: (robotID: string) => ({
+      endPoint: `/api/v1/robot/command/patrol/${robotID}`,
```

Comment:
You need a route ID for this endpoint now.
```suggestion
      endPoint: `/api/v1/robot/command/patrol/${robotID}/${routeID}`,
```